tests/test-cmd/test.lint.js

```diff
@@ -85,10 +86,15 @@ describe('lint', () => {
   it('passes through linter configuration', () => {
     const {lint, createLinter} = setUp();
     return lint({
+      // $FLOW_IGNORE: wrong type used for testing purpose
```

Comment:
This test should probably be re-written to pass in realistic values. It was asserting that a parameter sent to `lint()` gets passed to `createLinter()`. I guess my main concern was that passing in a default value could have caused a silent failure. In other words, if you pass in opposites of the defaults for each parameter then that should be enough to make the same assertion as before.